lib/livebook/notebook/explore/intro_to_livebook.livemd

```diff
@@ -88,8 +88,9 @@ You can explore all supported expressions [here](https://katex.org/docs/supporte
 Once you start using notebooks more, it's gonna be beneficial
 to optimise how you move around. Livebook leverages the concept of
 **navigation**/**insert** modes and offers many shortcuts for common operations.
-Make sure to check out the shortcuts by clicking the "Keyboard" icon in
-the sidebar or by typing `?`.
+Make sure to check out the shortcuts by clicking the "Keyboard" icon
+(<i class="ri-keyboard-box-fill text-xl align-middle"></i>) in the sidebar or
+by pressing <kbd>?</kbd>.
```

Comment:
Can you please add a similar change to the "elixir_and_livebook.livemd" when we mention `"Runtime" icon`?

Reply:
I'm on it, trying to have specific classes as you suggested!

Reply:
Updated, let me know if that's what you had in mind. Also, there are some formatting changes, because I pasted the contents of a saved notebook.